iron-overlay-behavior.html

```diff
@@ -341,9 +341,7 @@
       }
 
       this.fire('iron-overlay-opened');
-
-      this._squelchNextResize = true;
-      this.async(this.notifyResize);
+      this.async(this.refit);
```

Comment:
True. But when an overlay gets opened, it is not resized, it just needs to refit.
If we need to `notifyResize`, it might be that other listeners are invoked before the overlay.

What should happen is:
- overlay refits
- overlay fires notify resize
- other listeners are execute

If this order is not respected, other listeners might be executed too early, before the overlay gets a chance to refit.

WDYT? do we need to `notifyResize`?